src/main/java/ar/com/fdvs/dj/domain/DJCrosstabColumn.java

```diff
@@ -51,6 +52,10 @@
 
 	private String totalLegend;
 
+    private BucketOrder order = BucketOrder.ASCENDING;

```

Comment:
Please, do not add JasperReport specific classes into the domain model. 
Create a simple enum (name it something like `DJCrosstabBucketOrder`) and add the values ASCENDING, DESCENDING and NONE into it (as an example, you can see class `ImageScaleMode`)
Later, in the Dj2JrCrosstabBuilder you must convert it from the DJ world into the JR world.
I'm not sure this is the best, but it is how the rest of the library was done, we are not mixing now.